core/drivers/stm32_gpio.c

```diff
@@ -144,6 +144,14 @@ struct stm32_gpio_bank {
 	STAILQ_ENTRY(stm32_gpio_bank) link;
 };
 
+/*
+ * Compatibility infoamtion of supported banks
```

Comment:
Typo, I suppose you want to say "information"

Reply:
Right! thanks. checkpatch is not our friend here.

Reply:
You were too creative for checkpatch :wink: 
```
$ wget https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/plain/scripts/spelling.txt
$ grep information spelling.txt
infomation||information
informatiom||information
informations||information
informtion||information
infromation||information
```